### PR TITLE
French Traduction

### PR DESCRIPTION
**Pull request by modopro76**
# SCP:SL Translation PR Template

## Language
French

## Type of change
- [X] New addition
- [ ] Changes to current existing

## Have you tested the changes you've made?
- [X] Yes
- [ ] No

# Checklist:

- [X] I have made sure to not commit changes to other stuff than my own or the default english one
- [X] I have ran `EncodingNormalizer.exe` on my changed files before commiting
- [X] I have not made sure to not submit a duplicate translation language 
- [X] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

### CODE REVIEW

**Comment by Rnen** [comment]
Awaiting review by french speaking staff

File: Translations/French (Français) by DestinyGaming/MainMenu.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
+REJOINDRE
+FERMER
+RÉSOLUTION DE L'OMBRE
+OPTIONS
+BASÉ SUR <i>SCP: CONTAINMENT BREACH</i> | <i> DestinyGaming </i>
```
Comment (Rnen) [inline Translations/French (Français) by DestinyGaming/MainMenu.txt]: Do not include advertisements or pointers to non-game sources

Comment (modopro76) [reply Translations/French (Français) by DestinyGaming/MainMenu.txt]: Oh I remove it sorry I have copied my old translation

Comment (modopro76) [reply Translations/French (Français) by DestinyGaming/MainMenu.txt]: Hello,

I have changed what you want and I have checked all files and replace errors.

**Comment by Rnen** [comment]
(if you're done making changes, you need to re-request a review)

**Comment by modopro76** [comment]
Hello,
I can't re request a review.
Someone with write access has to validate the review.

**Comment by MichalPetryka** [comment]
Since we're aiming for one translation per language, could you make that we have only one French translation?

**Comment by modopro76** [comment]
Hello,

It is not a problem for me to update the translation at any game updates but I don't want to replace others translations. I just want to propose my translation to french players and they can choose my translation if they want.

**Comment by Rnen** [comment]
The ultimate goal of the game translations is to have **only 1** of each language. 
Some time in the future, all duplicate languages will be removed, possibly with the author of the old duplicate credited in a seperate file

**Comment by Rnen** [comment]
This has been declined due to beeing a duplicate submission, and it containing multiple problems listed above.
Please make changes to the current existing translation, rather than adding another version.

**Pull request closed without merging**